src/Renderer/PointsMaterial.js

```diff
@@ -201,15 +389,49 @@ class PointsMaterial extends THREE.RawShaderMaterial {
             data[j + 3] = visible ? parseInt(255 * opacity, 10) : 0;
         }
 
-        this.classificationLUT.needsUpdate = true;
+        const other = this.otherScheme;
```

Comment:
Could you please split `recomputeClassification` and `recomputeDiscrete`?
You could maybe share code in a third function called by those two.